images/win/scripts/Installers/Windows2019/Install-WSL1.ps1

```diff
@@ -0,0 +1,8 @@
+################################################################################
+##  File:  Install-WSL1.ps1
+##  Desc:  Install Windows Subsystem for Linux
+################################################################################
+
+Write-Host "Install Windows Subsystem for Linux"
+
+Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
```

Comment:
Will this override default bash that come from Git?

Reply:
No, wsl does not include any linux distribution (user should choose one) and even if it did the linux command need to be invoked specially `wsl ...`